install-steps.md

```diff
@@ -1,193 +1,821 @@
+   dhcp-option=3,<baremetal-net-gateway-ip>
+   #dhcp-option=3,10.0.1.254
+   resolv-file=/etc/resolv.conf.upstream
+   interface=<nic-with-access-to-baremetal-net>
+   #interface=em2
+   server=<ip-of-existing-server-on-baremetal-net>
+  
+
+   #Wildcard for apps -- make changes to cluster-name (openshift) and domain (example.com)
+   address=/.apps.openshift.example.com/<wildcard-ingress-lb-ip>
+
+   #Static IPs for Masters
+   dhcp-host=<NIC2-mac-address>,provisioner.openshift.example.com,<ip-of-provisioner>
+   dhcp-host=<NIC2-mac-address>,openshift-master-0.openshift.example.com,<ip-of-master-0>
+   dhcp-host=<NIC2-mac-address>,openshift-master-1.openshift.example.com,<ip-of-master-1>
+   dhcp-host=<NIC2-mac-address>,openshift-master-2.openshift.example.com,<ip-of-master-2>
+   dhcp-host=<NIC2-mac-address>,openshift-worker-0.openshift.example.com,<ip-of-worker-0>
+   dhcp-host=<NIC2-mac-address>,openshift-worker-1.openshift.example.com,<ip-of-worker-1>
+   ~~~
+5. Create the `resolv.conf.upstream` file in order to provide DNS fowarding to an existing DNS server for resolution to the outside world.
+   ~~~sh
+   search <domain.com>
+   nameserver <ip-of-my-existing-dns-nameserver>
+   ~~~
+6. Restart the `dnsmasq` service 
+   ~~~sh
+   systemctl restart dnsmasq
+   ~~~
+7. Verify the `dnsmasq` service is running.
+   ~~~sh
+   systemctl status dnsmasq
+   ~~~
+
+# Install RHEL on the Provision Node
+
+With the networking portions complete, the next step in installing the 
+Red Hat OpenShift (OCP) cluster is to install RHEL 8
+on the provision node. This node will be used as the orchestrator in installing
+the OCP cluster on the 3 master and 2 worker nodes. For the purposes of this
+document, installing RHEL on the provision node is out of scope. However, options
+include, but not limited to, using a RHEL Satellite server, PXE, or installation
+media. 
+
+# Preparing the Provision node for OpenShift Install
+
+The following steps need to be performed in order to prepare the environment. 
+
+1. Login into the provision node via `ssh`
+2. Start `firewalld`, enable the `http` service, enable port 5000.
+   ~~~sh
+   systemctl start firewalld
+   firewall-cmd --zone=public --add-service=http --permanent
+   firewall-cmd --zone=public --permanent --add-service=http --permanent
+   firewall-cmd --add-port=5000/tcp --zone=libvirt  --permanent
+   firewall-cmd --add-port=5000/tcp --zone=public   --permanent
+   firewall-cmd --reload
+   ~~~
+3. Disable SELinux
+    ~~~sh
+    sed -i "s/=enforcing/=permissive/g" /etc/selinux/config
+    setenforce 0
+    ~~~
+4. Register your environment using `subscription-manager`
+   ~~~sh
+   subscription-manager register --username=<user> --password=<pass>
+   subscription-manager repos --enable=rhel-8-for-x86_64-appstream-rpms --enable=rhel-8-for-x86_64-baseos-rpms
+   ~~~
+4. Install the following packages 
+   ~~~sh
+   dnf install -y libvirt qemu-kvm mkisofs python3-devel python3-pip gcc ipmitool
```

Comment:
pip and gcc are not needed

Reply:
OK - will make that change.

Reply:
confirmed dont need python3-pip or gcc

Reply:
require the package `jq` to be installed due to capturing the RHCOS_URI via command later in the document.